documentation/api/io.strimzi.api.kafka.model.KafkaConnectSpec.adoc

```diff
@@ -4,9 +4,18 @@ Use `KafkaConnectSpec` properties to configure a Kafka Connect cluster.
 .`logging`
 Kafka Connect has its own configurable loggers:
 
-* `connect.root.logger.level`
+* `log4j.rootLogger`
 * `log4j.logger.org.reflections`
 
+Further loggers are added dependently on running Kafka Connect plugins.
+
+To get complete list of connect loggers run from any Kafka broker pod:
+
+[source,curl,subs=attributes+]
+----
+curl -s http://<connect-cluster-name>-connect-api:8083/admin/loggers/
+----
+
 Kafka Connect uses the Apache `log4j` logger implementation.
 
 Use the `logging` property to configure loggers and logger levels.
```

Comment:
Do we need to say what levels can be used?

Reply:
Mm, maybe yes. We do list logger levels for example in https://github.com/strimzi/strimzi-kafka-operator/blob/master/documentation/modules/cruise-control/proc-deploying-cruise-control.adoc